### PR TITLE
Redirect surveillance.mozilla.org to foundation.mozilla.org.

### PR DESCRIPTION
**Pull request by smarnach**
## Refractr PR Checklist

JIRA ticket: https://jira.mozilla.com/browse/SE-1627

When creating a PR for Refractr, confirm you've done the following steps for smooth CI and CD experiences:
- [x] Have you updated the relevant YAML in the PR?
- [x] Have you checked if there are any TLS cert concerns - e.g. if the domain being redirected already exists, and it is being changed to point at Refractr, is a temporary TLS 'outage' while waiting for Lets Encrypt certification via HTTP challenge okay? If not, [have you followed these steps for using DNS challenges with our cert-manager setup](https://mana.mozilla.org/wiki/display/SRE/Refractr+-+How+To+-+DNS+Challenges)?
- [x] If desired, have you generated the Nginx manually to confirm addition works as expected? 
- [x] If desired, are you able to connect to EKS (cluster itse-apps-prod-1, namespace fluxcd) to more closely monitor the deploys?

After PR merge, next steps include:
- [ ] If going straight from main merge & Stage deploy to a release & production deploy, create the relevant GitHub release with an incremented version / tag applied.
- [x] Confirm you are ready and able to perform the requested DNS creation or change post-deploy?

### CODE REVIEW

**Comment by smarnach** [comment]
Build failed due to Docker Hub rate limiting.

**Comment by cmharlow** [comment]
on it.

**Review by cmharlow — COMMENTED** [review]
1 small change - would add the jira ticket (SE-1627) or the original bugzilla ticket as a comment above the added redirect. Just seems like a nice step to keep history in this file in case someone asks who and why a redirect was changed.

File: prod-refractr.yml [hunk 1]
```diff
@@ -142,6 +142,7 @@ refracts:
 - foundation.mozilla.org/:
   - mozillafoundation.com
   - mozillafoundation.net
+  - surveillance.mozilla.org
```
Comment (cmharlow) [inline prod-refractr.yml:145]:
```suggestion
  # SE-1627
  - surveillance.mozilla.org
```

**Review by cmharlow — CHANGES_REQUESTED** [review]

File: prod-refractr.yml (hunk 1 above)
Comment (cmharlow) [reply prod-refractr.yml:145]: adding this comment to go ahead and set up the deploy, since i need that to go out today. sorry, i'd normally wait!

**Review by cmharlow — APPROVED** [review]

**Pull request merged**